Logger.js

```diff
@@ -1,4 +1,4 @@
-import NUObject from 'service/NUObject';
+import NUObject from './NUObject';
```

Comment:
let's revert this! Since it's in the library we do not know how the lib is linked